WalletWasabi.Fluent/ViewModels/Dialogs/ShuttingDownViewModel.cs

```diff
@@ -1,31 +1,43 @@
+using ReactiveUI;
 using System.Reactive;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
 
 namespace WalletWasabi.Fluent.ViewModels.Dialogs;
 
-[NavigationMetaData(Title = "Wasabi is shutting down...")]
+[NavigationMetaData(Title = "Please wait to shut down...")]
 public partial class ShuttingDownViewModel : DialogViewModelBase<Unit>
```

Comment:
I just realized this is a `DialogViewModelBase`, I don't see why.

This is the only occurrence, and it could be a simple RoutableViewModel.
```
public void OnShutdownPrevented()
{
	RxApp.MainThreadScheduler.Schedule(async () =>
		await MainViewModel.Instance.CompactDialogScreen.NavigateDialogAsync(new ShuttingDownViewModel(this)));
}
```

Reply:
@soosr fixed in latest commit.